jwql/website/apps/jwql/migrations/0017_centralstorage_fgsanomaly_filesystemcharacteristics_and_more.py

```diff
@@ -0,0 +1,622 @@
+                ('used', models.FloatField()),
+                ('available', models.FloatField()),
+            ],
+            options={
+                'db_table': 'central_storage',
+                'db_table_comment': 'monitors',
+                'managed': True,
+            },
+        )
+        migrations.CreateModel(
+            name='FgsAnomaly',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('rootname', models.CharField()),
+                ('flag_date', models.DateTimeField()),
+                ('user', models.CharField()),
+                ('cosmic_ray_shower', models.BooleanField()),
+                ('diffraction_spike', models.BooleanField()),
+                ('excessive_saturation', models.BooleanField()),
+                ('guidestar_failure', models.BooleanField()),
+                ('persistence', models.BooleanField()),
+                ('crosstalk', models.BooleanField()),
+                ('data_transfer_error', models.BooleanField()),
+                ('ghost', models.BooleanField()),
+                ('snowball', models.BooleanField()),
+                ('other', models.BooleanField()),
+            ],
+            options={
+                'db_table': 'fgs_anomaly',
+                'db_table_comment': 'monitors',
+                'managed': True,
+            },
+        )
+        migrations.CreateModel(
+            name='FilesystemCharacteristics',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('date', models.DateTimeField()),
+                ('instrument', models.TextField()),
+                ('filter_pupil', models.TextField(blank=True, null=True)),
+                ('obs_per_filter_pupil', models.TextField(blank=True, null=True)),
+            ],
+            options={
+                'db_table': 'filesystem_characteristics',
+                'db_table_comment': 'monitors',
+                'managed': True,
+            },
+        )
+        migrations.CreateModel(
+            name='FilesystemGeneral',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('date', models.DateTimeField(unique=True)),
+                ('total_file_count', models.IntegerField()),
+                ('total_file_size', models.FloatField()),
+                ('fits_file_count', models.IntegerField()),
+                ('fits_file_size', models.FloatField()),
+                ('used', models.FloatField()),
+                ('available', models.FloatField()),
+            ],
+            options={
+                'db_table': 'filesystem_general',
+                'db_table_comment': 'monitors',
+                'managed': True,
+            },
+        )
+        migrations.CreateModel(
```

Comment:
I see some anomaly tables in here, they won't over write or reset anything when the migrations are performed, right?

Reply:
Correct. These are the existing anomaly tables, by instrument, in the JWQLDB database, and they won't have any effect on the by-exposure anomaly table in DJANGODB.

Reply:
Thank you!